docs/recipes/upgrading/v2-to-v3.md

```diff
@@ -0,0 +1,99 @@
+# How do I upgrade from SAFE v2 to v3?
+
+There have been a number of changes between the second and third major versions of the SAFE template. This guide shows you how to upgrade your v2 project to v3.
+
+> If you haven't done so already then you will need to install the prequisites listed in the [Quick Start](../../../quickstart) guide.
+
+### Terminology for this Recipe:
+
+* ***"Overwrite"***: Take the file from the "new" V3 template and copy it over the equivalent file in your existing project.
+* ***"Delete"***: Delete the file from your existing project. It is no longer required.
+* ***"Add"***: Add the file to your existing project. It is a new file added in V3.
+
+#### 1. Install the v3 template
+Download and install the latest SAFE Stack V3 template by running the following command:
+
+```bash
+dotnet new -i SAFE.Template
+```
+
+#### 2. Create a v3 project
+Create a new SAFE project in the `safetemp` folder. We will use this as a basis for our conversion.
+
+```fsharp
+dotnet new SAFE -o safetemp
+```
+
+#### 3. Branch your code
+We advise committing or stashing any unsaved changes to your code and making a new branch to perform the upgrade.
+
+You can then test it in isolation and then safely merge back in.
+
+#### 4. Update dotnet tools
+This file lists any custom dotnet tools used.
+
+* **Overwrite** the `.config/dotnet-tools.json` file.
+
+> **Important**! If you have installed extra dotnet tools, you will need to add them back in manually.
+
+#### 5. Update global.json 
+
+* **Overwrite** the `global.json` file
+
+#### 6. Update Paket dependencies
+* **Overwrite** the `paket.dependencies` file in the root of the solution.
+* **Overwrite** the `paket.lock` file.
+* **Overwrite** all `paket.references` files.
+
+**Important** If you have installed extra NuGet packages, you will need to add them back in manually to the dependencies and references files.
+
+Run paket to update project references:
+
+```bash
+dotnet paket install
+```
+
+* If you *have* added any extra NuGet packages, this command will also generate a new `paket.lock` file.
+
+#### 7. Update the npm dependancies 
+* **Overwite** the `package.json` file
+* **Overwite** the `package-lock.json` file
+
+**Important** If you have installed extra npm packages, you will need to add them back in manually to the dependencies.
+
```

Comment:
I could be tired, but it seems like a lot of steps. Would a script be a useful idea since it seems formulaic?